README.md

```diff
@@ -39,9 +39,9 @@ sudo apt-get install -y python-catkin-tools git
 
 ### Zivid SDK
 
-To use the ROS driver you need to download and install the "Zivid Core" package. SDK version 2.0 is supported.
-See [releases](https://github.com/zivid/zivid-ros/releases) for older ROS driver releases that supports
-older SDK versions.
+To use the ROS driver you need to download and install the "Zivid Core" package. Zivid SDK version 2.x is
```

Comment:
Should we actually say 2.x here? For instance, this commit of the zivid-ros driver will likely work with 2.2, but not necessary support new features that are added.

For me this is okey, just wanted to bring it up.

Reply:
Yeah, I was thinking to maybe say 2.0 and 2.1 for now. I can do that. It is probable that we may have a breakage in 2.2 or later due to adding new setting types (enum) which is not handled by the ROS settings generator as of now.

Reply:
Yes, I think also think say that we support 2.0 + 2.1.